supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,70 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Technical best practices
+
+* Include links only when necessary.
+* Use hyperlinks and use them consistently.
+* Do not use JavaScript links. 
+* Do not use Flash links.
+* Do not use links in Frames or iframes.
+* Default to followable, crawlable links. Do not use “nofollow” links unless absolutely necessary.
+
+
+.Link text best practices
+
+* Contextually describe what the user will find at the target location so that they can decide if they want to leave their current location. 
+* Avoid irrelevant link text.
+* You can use a sentence or sentence fragment as the link text.
```

Comment:
"You can" implies that there are other options. If you want people to use either a sentence (but long link strings are not recommended by IBM Style because of translation considerations) or sentence fragment, then drop the "you can."

Also, the "you can" makes this list not-parallel.